packages/ui-library/src/components/input-field-text/index.stories.ts

```diff
@@ -440,7 +440,7 @@ SizeVariant.story = {
 /**
  * ## Content / Settings
  * ### Type
- * The Input Field Text component can have all the types an html input can have, except the number and the unit type, which is covered in the Input Field Number component. For more information have a look at the [Input Field Number](/docs/design-system-web-components-forms-input-field-number--docs) component.
+ * The Input Field Text component can have all the types an html input can have, except the number and the unit type, which is covered in the Input Field Number component. For more information have a look at the [Input Field Number](/docs/components-forms-input-field-number--docs) component.
```

Comment:
The Input Field Number link is not working

Reply:
done

Reply:
Link still not working.

I think the `-forms` in the link needs to be removed as well. The links should be `/docs/components-input-field-number--docs`, if I am not mistaken.

Reply:
done